CHANGES.rst

```diff
@@ -1,3 +1,189 @@
+Ctapipe v0.18.0 (2023-02-09)
+============================
+
+
+API Changes
+-----------
+
+- ctapipe now uses entry points for plugin discovery. ``EventSource`` implementations 
+  now need to advertise a ``ctapipe_io`` entry point, to be discovered by ctapipe.
+  Additionally, ctapipe now includes preliminary support for discovering ``Reconstructor``
+  implementations via the ``ctapipe_reco`` entry_point. [`#2101 <https://github.com/cta-observatory/ctapipe/pull/2101>`__]
+
+- Migrate muon analysis into the ``ctapipe-process`` tool:
+
+  1. The former ``muon_reconstruction`` tool is dropped and all functionalities are transferred 
+     into the ``ctapipe-process`` tool.
+
+  2. The ``process`` tool now has a ``write_muon_parameters`` flag which defaults to ``false``.
+     Muons are only analyzed and written if the flag is set. Analyzing muons requires DL1b image 
+     parameters so even if ``write_parameters`` is ``false`` and no parameters are present in the 
+     ``EventSource`` those are computed. 
+
+  3. Two instances of ``QualityQuery``, ``MuonProcessor.ImageParameterQuery`` and ``MuonProcessor.RingQuery`` 
+     are added to the muon analysis to either preselect images according to image parameters and 
+     to select images according to the initial, geometrical ring fit for further processing. 
+     Deselected events or those where the muon analysis fails are being returned and written 
```

Comment:
I also dislike this sentence.
`Deselected events and those where the muon analysis fails, are written out as nan` ?

Reply:
changed to "filled with invalid value markers"